nixio/validate.py

```diff
@@ -146,7 +146,7 @@ def check_data_arrays(self, da, da_idx, blk_idx):
 
         if da.dimensions:
             for i, dim in enumerate(da.dimensions):
-                if dim.dimension_type == 'range':
+                if dim.dimension_type.value == 'range':
```

Comment:
Would it be more idiomatic to do `dim.dimension_type == DimensionType.Range`?

Reply:
In general, one should be able to change the value of the enum with no (or little) consequence.  In our case, the consequence is the value stored on disk would change (breaking the NIX format), but all comparisons would work.

Reply:
I guess we should add a comment to the enums as well, that the values of the enum are part of the NIX file format.

Reply:
Good idea!